streams/cn.m3u

```diff
@@ -1447,3 +1447,43 @@ http://yslk.chinashadt.com:1635/live/stream:di1.stream/playlist.m3u8
 http://yslk.chinashadt.com:1635/live/stream:di2.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="",龙岩综合 (540p)
 http://stream.lytv.net.cn/2/sd/live.m3u8
+#EXTINF:-1 tvg-id="",Ankang TV-1 News [Geo-blocked]
+http://124.47.34.186/PLTV/88888888/224/3221225855/index.m3u8
+#EXTINF:-1 tvg-id="",Beijing Traffic Radio TV [Geo-blocked]
+http://123.56.24.28:1935/live/fm1039/96K/tzwj_video.m3u8
+#EXTINF:-1 tvg-id="",Beijing Youth Radio [Geo-blocked]
+http://audiolive.rbc.cn:1935/live/am927/96K/tzwj_video.m3u8
+#EXTINF:-1 tvg-id="",Binchow News TV [Geo-blocked]
+http://stream.bzcm.net/2/sd/live.m3u8
+#EXTINF:-1 tvg-id="",Binchow Public TV [Geo-blocked]
+http://stream.bzcm.net/1/sd/live.m3u8
+#EXTINF:-1 tvg-id="",Binhai Life TV [Geo-blocked]
+http://60.30.52.41/live/bhtv2/playlist.m3u8
+#EXTINF:-1 tvg-id="",Binhai News TV [Geo-blocked]
+http://60.30.52.41/live/bhtv1/playlist.m3u8
+#EXTINF:-1 tvg-id="",Fengtai Culture & Life TV
+http://60.175.115.119:1935/live/wenhua/playlist.m3u8
+#EXTINF:-1 tvg-id="",Fengtai News TV
+http://60.175.115.119:1935/live/zonghe/playlist.m3u8
+#EXTINF:-1 tvg-id="",Foochow Children TV
+http://live.zohi.tv/video/s10001-sepd-4/index.m3u8
```

Comment:
As for Zohi : http://live.zohi.tv/video/s10001-fztv-1/index.m3u8
http://live.zohi.tv/video/s10001-fztv-3/index.m3u8
http://live.zohi.tv/video/s10001-fztv-4/index.m3u8

Radio : http://live.zohi.tv/audio/s10001-FM944/index.m3u8 Fuzhou (?) News Radio
http://live.zohi.tv/audio/s10001-FM893/index.m3u8 Fuzhou Music Radio
http://live.zohi.tv/audio/s10001-FM876/index.m3u8 Fuzhou Traffic Radio
http://live.zohi.tv/audio/s10001-FM901/index.m3u8 Fuzhou ? Radio